### PR TITLE
Minor typo/formatting fixes

### PR DESCRIPTION
**Pull request by sebr**

### CODE REVIEW

File: src/language-service/src/schemas/types.ts [hunk 1]
```diff
@@ -182,7 +182,7 @@ export type Template = string;
 /**
  * @TJS-pattern ^(?:[01]\d|2[0123]):(?:[012345]\d):(?:[012345]\d)$
  */
-export type Time = string;
+export type Time = string | string[];
```
Comment (frenck) [inline src/language-service/src/schemas/types.ts:185]: We need to specify the regex for the items in that case:

```
* @items.pattern ^(?:[01]\d|2[0123]):(?:[012345]\d):(?:[012345]\d)$
```

However, this is too generic. Not every part of the whole HA schema accepts multiple items in `Time` by default. This applies to the automation trigger only.

File: .github/ISSUE_TEMPLATE/1. yaml-error-or-warning.md [hunk 2]
```diff
@@ -14,13 +14,13 @@ labels: 'yaml'
     the raw YAML. If it's a generic YAML Parser error, please report it here:
     https://github.com/JPinkney/yaml-ast-parser
 
-    In doubt? Read more: 
+    In doubt? Read more:
```
Comment (frenck) [inline .github/ISSUE_TEMPLATE/1. yaml-error-or-warning.md:17]: These kinds of improvements are awesome!

But do them in a separate PR. I've could have merged these, but are now blocked by a comment in the actual code of this PR.

**Review by frenck — CHANGES_REQUESTED** [review]

File: src/language-service/src/schemas/types.ts (hunk 1 above)
Comment (frenck) [reply src/language-service/src/schemas/types.ts:185]: I've created #552 to replace this part, as the upcoming Home Assistant 0.115 will have even more changes to this.

So could you revert the pattern/schema changes? So we can merge all other stuff in this PR? 👍

Comment (sebr) [reply src/language-service/src/schemas/types.ts:185]: SGTM thanks @frenck

**Comment by sebr** [comment]
@frenck I've removed the typing change so this PR now addresses nit-pick grammar/formatting fixes.

**Review by frenck — APPROVED** [review]
Thanks, @sebr! Looks good! 👍
